framework/include/base/MooseDefaultConfig.h

```diff
@@ -15,7 +15,7 @@
 
 /* The size of the derivative backing array */
 #ifndef MOOSE_AD_MAX_DOFS_PER_ELEM
-#define MOOSE_AD_MAX_DOFS_PER_ELEM 50
+#define MOOSE_AD_MAX_DOFS_PER_ELEM 53
```

Comment:
I ll check but I think I needed a little more with temperature in the mix too.

Reply:
For a 3D case for sure, but the test I had to adjust this for is a wcnsfv problem